src/store/wallet/wallet-selectors.ts

```diff
@@ -122,13 +122,16 @@ export const useCollectiblesListSelector = () => {
   return useMemo(() => assetsList.filter(asset => isCollectible(asset)), [assetsList]);
 };
 
-export const useVisibleCollectiblesListSelector = () => {
+export const useNonZeroBalanceCollectiblesListSelector = () => {
   const collectiblesList = useCollectiblesListSelector();
```

Comment:
if `useCollectiblesListSelector` is used only here remove `useNonZeroBalanceCollectiblesListSelector` and move it's logic inside `useCollectiblesListSelector`